### ISSUE TITLE
Change validation error for idea title

### ISSUE THREAD

**Issue by rittermo** [liqd/aula#500]
Entering a special character like “!,.@” in an idea title produces a validation error that reads “unerwartet '.'erwartet letter or digit, space oder end of input”. The validation message should be changed to “Sonderzeichen wie „[unallowed character]“ sind im Titel der Idee nicht erlaubt.”

---
Original text: 
Erstellung von neuer Idee "Mehr Stuff!!": Scheitert mit Fehler "unerwartet '!'erwartet letter or digit, space oder end of input"
Fehler erscheint unhervorgehoben oben im Dokument, wodurch ich ihn nicht sofort entdeckt habe und erst einmal verwirrt war, wieso es nicht klappte.

**Comment by fisx** [comment]
(we should probably fix this for all occurrances, not only the ones that we encounter during testing.)

**Comment by rittermo** [comment]
@fisx In v0.13.0 the error message seems unchanged. Would it be easier if we look at the wording of all validation errors as a whole for iteration 2?

**Comment by fisx** [comment]
@rittermo ok, I'm re-opening this and moving it to it.2.

These messages are generated with a template from case-specific sentence fragments, so to get precisely the wording you suggested would be awkward and I was hoping it's not necessary.  Probably better to talk about this in person?

**Comment by rittermo** [comment]
@fisx Yes, sure. I’m sure we can find a good solution with the existing templates. Let’s do this in person towards the end of next week.

**Comment by fisx** [comment]
we need to come up with a way to take out all german text for review by editors.  it'll be interesting to extend whatever system we can come up with to the generated errors here.

**Comment by andorp** [comment]
Related to: https://github.com/liqd/aula/pull/752

**Comment by fisx** [comment]
related: #788.
